doc/source/cookbook.rst

```diff
@@ -1318,3 +1318,80 @@ of the data values:
        'weight': [100, 140, 180],
        'sex': ['Male', 'Female']})
    df
+
+Categorical Data
+----------------
+
+`Using categorical data type to store data more efficiently and consistently in multiple DataFrames
+
+`For more information about categorical data see <http://pandas.pydata.org/pandas-docs/stable/categorical.html>`
+
+Create two DataFrames with one column each from a subset of the unique values with overlap. Append the dataframes and examine the memory usage.
+
+.. ipython:: python
+
+   from pandas.api.types import CategoricalDtype
+
+   domain = np.array(['A','B','C','D','E','F','G','H','I','J','a','b','c','d','e','f','g','h','i','j'])
+
+   df1 = pd.DataFrame({'A': domain.take(np.random.randint(0,12,size=100000))})
+   df2 = pd.DataFrame({'A': domain.take(np.random.randint(8,20,size=100000))})
+   print('df1.A Data Type:', df1.A.dtype)
```

Comment:
you don't need to use print, the output is printed. 

you can use comments to indicate what's important

Reply:
will do.